2025-07-15 10:43:17,385 - INFO - ====== 开始运行实验 5 ======
2025-07-15 10:43:22,421 - INFO - Connected to Web3. Default account: 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
2025-07-15 10:43:22,570 - ERROR - 实验 5 运行失败: name 'get_certificate_factory' is not defined
Traceback (most recent call last):
  File "/Users/yangzongyou/Downloads/iccip 2022/code_exp/scripts/temp_exp5.py", line 96, in <module>
    run()
  File "/Users/yangzongyou/Downloads/iccip 2022/code_exp/scripts/temp_exp5.py", line 84, in run
    cert_factory = get_certificate_factory()
NameError: name 'get_certificate_factory' is not defined
